### ISSUE TITLE
[TBC classic] Quest "Tuskin' Raiders"  gives lua errors on acceptance, and everytime you load into a new zone

### ISSUE THREAD

**Issue by jao** [Questie/Questie#3389]
## Description

After getting the quest, and everytime I log back in I get this error:
```
Interface\AddOns\Questie\Modules\Map\QuestieMap.lua:546: No UiMapID or fitting parentAreaId for areaId : 3805 - Amani'shi Warbringer
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:1028: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:1026>
[string "=[C]"]: in function `error'
[string "@Interface\AddOns\Questie\Modules\Map\QuestieMap.lua"]:546: in function `DrawWorldIcon'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:966: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:913>
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:794: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:730>
[string "=[C]"]: in function `xpcall'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:1026: in function <...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:1024>
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:1079: in function `PopulateObjectiveNotes'
[string "@Interface\AddOns\Questie\Modules\Quest\QuestieQuest.lua"]:544: in function `GetAllQuestIds'
[string "@Interface\AddOns\Questie\Modules\QuestieInit.lua"]:152: in function <Interface\AddOns\Questie\Modules\QuestieInit.lua:50>

Questie: [ERROR] [QuestieQuest]: v6.8.6 - There was an error populating objectives for Tuskin' Raiders 11164 1 No error
```

## Wowhead link

https://tbc.wowhead.com/quest=11164/tuskin-raiders

## Questie & WoW version

Questie v6.8.6
TBC client after Zul'Aman patch

**Comment by Cabro** [comment]
Add in Questie\Database\Zones\zoneTables.lua line 197:

    [3805] = {{3433, 82.1, 64.2}},

**Comment by Muehe** [comment]
Fixed in 7f1b5677ae4c31f3636b6b1f7c92a8b3a318bae2

Release will be uploaded shortly™.
